<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LanguageTools</class>
 <widget class="qMRMLWidget" name="LanguageTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>427</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <widget class="QPushButton" name="updateButton">
     <property name="toolTip">
      <string>Download, compile, and install latest translations</string>
     </property>
     <property name="text">
      <string>Update translation files</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QTextEdit" name="statusTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="ctkCollapsibleButton" name="settingsCollapsibleButton">
     <property name="text">
      <string>Settings</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Qt 'lrelease' tool path:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ctkPathLineEdit" name="lreleasePathLineEdit">
        <property name="toolTip">
         <string>Path to Qt lrelease executable.</string>
        </property>
        <property name="showHistoryButton">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="slicerVersionLabel">
        <property name="text">
         <string>Slicer version:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="slicerVersionEdit">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;master, 4.13, 4.14, ...&lt;/p&gt;&lt;p&gt;Only used for GitHub and Crowdin multi-language.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>master</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="ctkCollapsibleButton" name="outputsCollapsibleButton">
     <property name="text">
      <string>Input translations</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="6" column="0">
       <widget class="QLabel" name="gitRepositoryLabel">
        <property name="text">
         <string>Repository:</string>
        </property>
        <property name="indent">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="ctkPathLineEdit" name="crowdinZipFilePathLineEdit">
        <property name="toolTip">
         <string>Zip file downloaded from https://crowdin.com/project/slicer using 'Build and Download'.</string>
        </property>
        <property name="settingKey">
         <string>Internationalization/LanguageTools/CrowdinZipFilePathHistory</string>
        </property>
        <property name="nameFilters">
         <stringlist>
          <string>*.zip</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QRadioButton" name="crowdinTsFolderRadioButton">
        <property name="text">
         <string>Crowdin .ts files folder (manual download)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="crowdinZipFileLabel">
        <property name="text">
         <string>Input zip file:</string>
        </property>
        <property name="indent">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="githubRepositoryEdit">
        <property name="toolTip">
         <string>Github repository that stores the translated strings.</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="crowdinTsFolderLabel">
        <property name="text">
         <string>Input folder:</string>
        </property>
        <property name="indent">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QRadioButton" name="crowdinZipFileRadioButton">
        <property name="text">
         <string>Crowdin multi-language (manual download, administrators only)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QRadioButton" name="githubSourceRadioButton">
        <property name="text">
         <string>GitHub (auto download)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ctkPathLineEdit" name="crowdinTsFolderPathLineEdit">
        <property name="toolTip">
         <string>Zip file downloaded from https://crowdin.com/project/slicer using 'Build and Download'.</string>
        </property>
        <property name="filters">
         <set>ctkPathLineEdit::Dirs|ctkPathLineEdit::Drives|ctkPathLineEdit::Executable|ctkPathLineEdit::NoDot|ctkPathLineEdit::NoDotDot|ctkPathLineEdit::Readable</set>
        </property>
        <property name="settingKey">
         <string>Internationalization/LanguageTools/CrowdinSingleLanguageFolderPathHistory</string>
        </property>
        <property name="nameFilters">
         <stringlist>
          <string>*</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="latestTsFileOnlyLabel">
        <property name="text">
         <string>Latest file only:</string>
        </property>
        <property name="indent">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="latestTsFileOnlyCheckBox">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If this option is checked then only the latest .ts file in the folder will be used. Enable this option to ignore previous files downloaded from crowdin that are saved in the same folder.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string> </string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="restartButton">
     <property name="toolTip">
      <string>Download, compile, and install latest translations</string>
     </property>
     <property name="text">
      <string>Restart the application</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QWidget</extends>
   <header>ctkPathLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
